composer.json

```diff
@@ -221,7 +219,8 @@
                 "Allow also YouTube embed URLs. Improve the YT regexp @see https://www.drupal.org/node/2899093": "https://www.drupal.org/files/issues/2899093-8.patch"
             },
             "drupal/views_data_export": {
-                "Display schema should extend rest export @see https://www.drupal.org/project/views_data_export/issues/2896104": "https://www.drupal.org/files/issues/2896104-2-fix_display_schema.patch"
+                "Display schema should extend rest export @see https://www.drupal.org/project/views_data_export/issues/2896104": "https://www.drupal.org/files/issues/2896104-2-fix_display_schema.patch",
+                "Download link returns 406 with Drupal ^8.5.0 @see https://www.drupal.org/project/views_data_export/issues/2951185": "https://www.drupal.org/files/issues/2951185-2.patch"
```

Comment:
This issue is a temporary workaround that adds the ?_format=csv query parameter to the link by default.
If the link is altered manually though, a critical error will still occur but by the module. So not an exception, but the page shows the text "Critical error" in the normal theme.

Reply:
That if you go to a page that the csv link is listed somewhere in the page, it is generated with the `?_format=csv` parameter. But you can always try to enter the url manually in the address bar without the parameter if you know it.

Reply:
Then you "don't know it" :) Because, now, the URL should contain the query param too.